DataConnectors/WireXSystems/WireXsystemsNFP.json

```diff
@@ -0,0 +1,119 @@
+{
+    "id": "WireX_Systems_NFP",
+    "title": "WireX Network Forensics Platform",
+    "publisher": "WireX_Systems",
+    "descriptionMarkdown": "The WireX Systems data connector allows security professional to integrate with Azure Sentinel to allow you to further enrich your forensics investigations; to not only encompass the contextual content offered by WireX but to analyze data from other sources, and to create custom dashboards to give the most complete picture during a forensic investigation and to create custom workflows.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Record_Recieved",
+            "baseQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"WireX Systems\"\n| where DeviceProduct == \"WireX NFP\"\n"
+        }
+    ],
+    "sampleQueries": [
```

Comment:
The sample queries are syntactically incorrect. I am sharing one updated as an example. Please update others too. Also would be great if you'd please validate each of these sample queries in Azure Sentinel Log Analytics editor on the CEF logs you got into Azure Sentinel or you can load the JSON file per step 3 of https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/ReadMe.md#build-the-connector and also click on the Run button in the Next steps tab of the data connector where each of these queries will show as a customer would see them. 
Updated example:
{
            "description" : "All Imported Events from WireX",
            "query": "CommonSecurityLog| where DeviceVendor == "WireX Systems"| where DeviceProduct == "WireX NFP""
        },

Reply:
I see that i ran things backwards, i made the query work on the demo systems and then cut and paste it into the .json files. I tried to make these changes that you suggested above on just query 1 and i could not get it to load. When I tried to make changes to it so that it would load 
"query": "CommonSecurityLog| where DeviceVendor == WireX| where DeviceProduct == WireX NFP"
i got failed: 'where' operator: Failed to resolve column or scalar expression named 'WireX'
If issue persists, please open a support ticket. Request id: 17f6ecec-8895-458c-9132-81970115c251

however in the demo systems contoso my original query worked.
CommonSecurityLog​
| where DeviceVendor in ("WireX")
| sort by TimeGenerated 

any suggestions.